jack-and-the-beans/buddies-ios
# Store filter settings in firebase

BUD-41

To Test:
- Save a filter, and make sure it persists to other views (after saving)
- Save a filter, close the app and make sure it is remembered.
